workspace.json

```diff
@@ -938,6 +938,14 @@
             "passWithNoTests": true
           },
           "outputs": ["coverage/libs/api/domains/driving-license"]
+        },
+        "schemas/external-openapi-generator": {
+          "builder": "@nrwl/workspace:run-commands",
+          "options": {
+            "commands": [
```

Comment:
It can be simplify to 

```
"command": "nx run clients-driving-license:schemas/openapi-generator"
```

Reply:
Actually, the following 3 commands are not necessary if you rename 

```
        "schemas/openapi-generator": {
          "builder": "@nrwl/workspace:run-commands",
          "options": {
            "command": "yarn openapi-generator -o libs/clients/driving-license/gen/fetch -i libs/clients/driving-license/src/openapi_doc.json"
          }
        }
```

to 

```
        "schemas/external-openapi-generator": {
          "builder": "@nrwl/workspace:run-commands",
          "options": {
            "command": "yarn openapi-generator -o libs/clients/driving-license/gen/fetch -i libs/clients/driving-license/src/openapi_doc.json"
          }
        }
```

then it will be executed at the first step of the schema generation and it will be available for the 3 API domains later on

Reply:
ah, ok, i'll try that